contracts/EverlongStrategy.sol

```diff
@@ -0,0 +1,482 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
```

Comment:
One suggestion I'd have going forward is to use `git mv` to rename files. It would be a lot easier to review this if I could see the diff between `contracts/Everlong.sol` and `contracts/EverlongStrategy.sol` in the Github UI